docs/source/notebooks/SMC2_gaussians.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
```

Comment:
<ul><li>The final samples, <em>i.e</em> those stored in the <code>trace</code>, will be taken exclusively from the final stage (β=1), i.e. the <strong>inferred</strong> <s>true</s> posterior <strong>-- which we hope is the true posterior distribution.</strong></li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc3/pull/3981/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
I mean "true" in the mathematical sense. I will clarify this.